Musescore.config

```diff
@@ -0,0 +1,2 @@
+// Add predefined macros for your project here. For example:
+// #define THE_ANSWER 42
```

Comment:
typo, sorry, I meant the one before 57d3883. And yes, your changes should be safe, as Long as you don't add a --hard